src/components/MoviesList/MoviesList.tsx

```diff
@@ -4,12 +4,23 @@ import { MovieCard } from '../MovieCard';
 
 interface Props {
   movies: Movie[];
+  query: string;
 }
 
-export const MoviesList: React.FC<Props> = ({ movies }) => (
-  <div className="movies">
-    {movies.map(movie => (
-      <MovieCard key={movie.imdbId} movie={movie} />
-    ))}
-  </div>
-);
+export const MoviesList: React.FC<Props> = ({ movies, query }) => {
+  const queryTolowerCase = query.toLowerCase();
+  const visibleMovies = movies.filter(movie => {
+    const { description, title } = movie;
+
+    return title.toLowerCase().includes(queryTolowerCase)
+      || description.toLowerCase().includes(queryTolowerCase);
+  });
```

Comment:
it's better to save movies in the state and remove visibleMovies variable. After filtering just update the state with movies. You can create state with movies in App component